pkg/cl/api.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package cl
```

Comment:
The `cl` package should not be at the root of the `pkg` hierarchy.

Reply:
Would this `cl` package be similar to `jwt`, `verifiable`, ... under the pkg/doc hierarchy?

(e.g., `pkg/doc/cl`)?

Reply:
Almost, it's more like high-level CL crypto services, not strongly attached to credentials/docs. But I guess it's ok to move them to doc. 
btw, our next step is to implement indy credential attachments which will definitely go to `pkg/doc`

Reply:
@troyronda done